velox/vector/tests/DecodedVectorTest.cpp

```diff
@@ -79,8 +80,17 @@ class DecodedVectorTest : public testing::Test, public VectorTestBase {
         }
         const bool isNull = (expected[index] == std::nullopt);
         if (dbgPrintVec) {
-          LOG(INFO) << "[" << index << "]:"
-                    << (isNull ? "NULL" : folly::to<std::string>(actualValue));
+          if constexpr (
+              std::is_same_v<T, UnscaledLongDecimal> ||
```

Comment:
Can reduce code by using a local variable to store the to_string result and then invoking LOG